RecoTracker/LST/src/alpaka/ES_ModulesDev.cc

```diff
@@ -0,0 +1,10 @@
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/typelookup.h"
+
+// Temporary hack: The DevHost instantiation is needed in both CPU and GPU plugins,
+// whereas the (non-host-)Device instantiation only in the GPU plugin
+TYPELOOKUP_DATA_REG(SDL::LSTESHostData<SDL::Dev>);
+TYPELOOKUP_DATA_REG(SDL::LSTESDeviceData<SDL::DevHost>);
```

Comment:
These should be placed directly into `RecoTracker/LST/src`

Reply:
is the comment only about `SDL::LSTESDeviceData<SDL::DevHost>` or both?
I'm trying to understand how `src` could depend on code defined in `alpaka`; is it an OK dependence?

Reply:
> is the comment only about `SDL::LSTESDeviceData<SDL::DevHost>` or both?

Both.

> I'm trying to understand how `src` could depend on code defined in `alpaka`; is it an OK dependence?

No, `{interface,src}` may not depend on code defined in `{interface,src}/alpaka`. But code templated over e.g. device type (or, code that does not depend on `ALPAKA_ACCELERATOR_NAMESPACE` in any way) can be placed in `{interface,src}`, and such templates can be instantiated for `alpaka::DevCpu` / `alpaka_common::DevHost` in the `{interface,src}` as well.